### ISSUE TITLE
Smartphone snippets spamming when smartphone flashlight are activated.

### ISSUE THREAD

**Issue by gettingusedto** [CleverRaven/Cataclysm-DDA#64858]
### Describe the bug

When activating the flashlight, the smartphone snippet thing repeatingly popping up.

### Attach save file

N/A

### Steps to reproduce

1. get a smartphone
2. check the snippet on the phone (not sure if its required)
3. turn on smartphone flashlight
4. snippet spam

### Expected behavior

Not spamming it.

### Screenshots

![Screenshot_20230405-201815](https://user-images.githubusercontent.com/78019001/230094038-0b246ccb-a306-4079-a637-f456c2193c74.jpg)

### Versions and configuration

- OS: Android
    - OS Version: Manufacturer: samsung; Model: SM-A710F; Release: 7.0; Incremental: A710FXXU2CRC2; 
- Game Version: ad4e5c9 [32-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Bombastic Perks [bombastic_perks],
    Xedra Evolved [xedra_evolved],
    Magiclysm [magiclysm]
]

**Comment by estebandellasilva** [comment]
can confirm its annoying

**Comment by juxe** [comment]
can confirm on latest experimental version this is happening to me. I was playing yesterday on the previous latest experimental version and did not encounter this issue.

1. start new game as survivor (starts with smartphone)
2. turn on smartphone flashlight
3. craft/practice Athletics (beginner)

attached is a screenshot I just took from my game where I encountered this
<img width="1920" alt="2023-04-05 14_49_43-Cataclysm_ Dark Days Ahead - de40ba8" src="https://user-images.githubusercontent.com/143510/230193320-be750405-e008-4e59-8a3c-03eda9c4792c.png">
